rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -92,7 +96,15 @@
           auth-id: "github_account_rpc_jenkins_svc"
           status-context: 'CIT/artifact-{image}-{context}'
     dsl: |
-      // CIT Slave node
+      node('ArtifactBuilder2') {{
+        dir("rpc-gating") {{
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            common = load 'pipeline-steps/common.groovy'
+            aio_prepare = load 'pipeline-steps/aio_prepare.groovy'
```

Comment:
Yeah, we need the rpc-openstack repo to be there. I've submitted https://github.com/rcbops/rpc-gating/pull/114 to break out the git repo prep as that's all we want for all the artifact build jobs.